src/pyrqlite/connections.py

```diff
@@ -43,7 +43,7 @@ class Connection(object):
     def __init__(self, host='localhost', port=4001,
                  user=None, password=None, connect_timeout=None,
                  detect_types=0, max_redirects=UNLIMITED_REDIRECTS,
-                 always_transactions=True):
+                 always_transactions=True, default_paramstyle='qmark'):
```

Comment:
In my remark about default_paramstyle in sqlalchemy's SQLiteDialect, my intention was to suggest that the ORM conform to the paramstyle of the database. The existing convention is for paramstyle to be constant for a particular database type, see:

https://www.python.org/dev/peps/pep-0249/#paramstyle